### PR TITLE
add example ssl attributes and mention other config attributes

### PR DESCRIPTION
**Pull request by alexpop**

### CODE REVIEW

**Comment by robbkidd** [comment]
Thanks for this, @alexpop! I'm double-checking the attributes used deeper the omnibus layers.

File: README.md [hunk 1]
```diff
@@ -26,11 +26,20 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
...
 ```
 
+If you wish to use custom SSL certificates, define the following `config` attributes:
+
+```ruby
+default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/certificate.crt'
+default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/certificate.key'
+```
+
+To see all supermarket `config` attributes you can override, see `/etc/supermarket/supermarket-running.json`
```
Comment (irvingpop) [inline README.md]: Hey @alexpop I recommend you link to this instead:  https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb

**Comment by alexpop** [comment]
Link updated, thanks for the feedback Irving.

**Comment by irvingpop** [comment]
:+1:

File: README.md [hunk 2]
```diff
@@ -26,11 +26,20 @@ default['supermarket_omnibus']['package_repo'] = 'chef-current'
...
 ```
 
+If you wish to use custom SSL certificates, define the following `config` attributes:
+
+```ruby
+default['supermarket_omnibus']['config']['ssl']['certificate'] = '/full/path/to/ssl.crt'
+default['supermarket_omnibus']['config']['ssl']['certificate_key'] = '/full/path/to/ssl.key'
+```
+
+To find out all supermarket `config` attributes you can override, see [omnibus-supermarket](https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb). An attribute like `default['supermarket']['nginx']['log_rotation']['num_to_keep'] = 10` translates to  `supermarket-omnibus-cookbook` as `default['supermarket_omnibus']['config']['nginx']['log_rotation']['num_to_keep'] = 10`
```
Comment (robbkidd) [inline README.md]: Being new to the wrapper of wrappers, it took me a little while to trace this. What do you think of a little expansion of this portion of the README?:

--

To find out all supermarket `config` attributes you can override, see [omnibus-supermarket](https://github.com/chef/omnibus-supermarket/blob/master/cookbooks/omnibus-supermarket/attributes/default.rb). Translation of attributes from `supermarket-omnibus-cookbook` to attributes in `omnibus-supermarket` occurs in the `supermarket_server` resource provided by this cookbook which produces a JSON that `omnibus-supermarket` reads.

```ruby
# an attribute you define via this supermarket_omnibus_cookbook
default['supermarket_omnibus']['config']['nginx']['log_rotation']['num_to_keep'] = 10

# becomes the following in omnibus_supermarket
default['supermarket']['nginx']['log_rotation']['num_to_keep'] = 10
```

**Comment by alexpop** [comment]
I agree Robb, you improvement is included now

**Comment by robbkidd** [comment]
:+1: Thanks!

**Pull request merged**